proto/decentraland/social/friendships/friendships.proto

```diff
@@ -6,14 +6,14 @@
 syntax = "proto3";
 package decentraland.social.friendships;
 
-// Validation and anti-spam errors
-enum FriendshipErrorCode {
-  TOO_MANY_REQUESTS_SENT = 0;
-  NOT_ENOUGH_TIME_PASSED = 1;
-  BLOCKED_USER = 2;
-  NON_EXISTING_USER = 3;
-  INVALID_REQUEST = 4;
-  UNKNOWN = 5;
+enum ServiceErrors {
+  UNKNOWN = 0;
+  BAD_REQUEST = 400;
+  UNAUTHORIZED = 401;
+  FORBIDDEN = 403;
+  NOT_FOUND = 404;
+  TOO_MANY_REQUESTS = 429;
```

Comment:
Do we want to rate limit the amount of messages sent by client?

Reply:
Nope, but synapse does, so I thought it may be convenient to add it, so we can map it to ours. wdyt? should I leave it, or is it no worthy?

Reply:
Ohh, okay, replicating synapse response should be okay then